includes/data-stores/class-wc-order-data-store-cpt.php

```diff
@@ -684,11 +624,16 @@ protected function get_wp_query_args( $query_vars ) {
 			}
 		}
 
+		if ( isset( $query_vars['customer'] ) && '' !== $query_vars['customer'] ) {
```

Comment:
@mikejolley Apparently not. :) [Looks like 0 is a valid param](https://github.com/woocommerce/woocommerce/blob/master/tests/unit-tests/order/functions.php#L166). I've switched it to check for empty string and empty array, though.